torch/testing/_internal/common_methods_invocations.py

```diff
@@ -529,6 +529,55 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
                             dim=(0, 1))))
         return inputs
 
+def sample_inputs_add(op_info, device, dtype, requires_grad, **kwargs):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
```

Comment:
`... *, for_inplace_variant=False, **kwargs)` in signature should be more straightforward